item/paintings.lua

```diff
@@ -135,6 +135,22 @@ function M.UseItem(User, SourceItem)
             "The picture begins to glow slightly and one can see that the potion inside the cauldron turns a dark green and an evil�skull�rises up out of the cauldon.")
         return
     end
+	local pictureno=SourceItem:getData("dickerquest");
+	if tonumber(pictureno) and User:getQuestProgress(674) == 100 then
+		if bit.band (User:getQuestProgress(676),tonumber(picturenumber)) == 0 then
+			User:setQuestProgress(676) = User:getQuestProgress(676) + tonumber(picturenumber)
```

Comment:
Danke,

 

Ich hab’s korrigiert und comitted. Alle 3 Fehler

 

Banduk

 

Von: Martin Karing [mailto:notifications@github.com] 
Gesendet: Montag, 19. Oktober 2015 23:37
An: Illarion-eV/Illarion-Content
Cc: Baylamon
Betreff: Re: [Illarion-Content] Quest Alberto Dicker (#25)

 

In item/paintings.lua <https://github.com/Illarion-eV/Illarion-Content/pull/25#discussion_r42429259> :

> @@ -135,6 +135,22 @@ function M.UseItem(User, SourceItem)
>              "The picture begins to glow slightly and one can see that the potion inside the cauldron turns a dark green and an evil�skull�rises up out of the cauldon.")
>          return
>      end
> +     local pictureno=SourceItem:getData("dickerquest");
> +     if tonumber(pictureno) and User:getQuestProgress(674) == 100 then
> +            if bit.band (User:getQuestProgress(676),tonumber(picturenumber)) == 0 then
> +                    User:setQuestProgress(676) = User:getQuestProgress(676) + tonumber(picturenumber)

You can't assign value to a function like that. The correct usage of setQuestProgress is: setQuestProgress(<questId>, <newValue>)

—
Reply to this email directly or view it on GitHub <https://github.com/Illarion-eV/Illarion-Content/pull/25/files#r42429259> .Das Bild wurde vom Absender entfernt.